doc/ni_by_example_2.md

```diff
@@ -125,7 +125,7 @@ Here's what you need to know:
   * `(lambda (u v) (u + v))` yields a closure. There's a lot of ink spilled on what a closure is and does. For the purpose of this tutorial, you can think of a closure as a a function that has not been supplied its arguments.
```

Comment:
maybe they don't realize how much people on the other side are cringing that there are two concepts that are almost interchangeable but different enough that it makes them cringe